02-household-queries/qt_summaries-gpt-3.5-turbo-instruct-k_4.json

```diff
@@ -0,0 +1,851 @@
+[
+    {
+        "id": 1,
+        "question": "The client has her 19 year old grandchild living with her. She says she does her own thing and they don't really share food. Does the grandchild need to be included?",
+        "guru_cards": [
+            "Who are mandatory HH members for food stamps?"
+        ],
+        "recall": 1.0,
+        "extra_cards": 3,
+        "derived_questions": [
```

Comment:
All of the questions derived by the LLM from the user's original question

Reply:
`derived_questions` in this file were generated using `openhermes`